workspaces/adr/.changeset/healthy-horses-wait.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage-community/plugin-adr-backend': minor
```

Comment:
as I said in the other comment we should try to avoid the breaking change, making this as a patch

Reply:
Hi @vinzscam . I've made the changes again a patch for the backend module and added the deprecation for all the types now. I think this should be ready to go now :)